docs/user-guide/rug/tests.md

```diff
@@ -4,26 +4,88 @@ rapid, in-memory testing of Rugs.
 
 [bdd]: https://en.wikipedia.org/wiki/Behavior-driven_development (Behavior-Driven Development)
 
-The framework is based on the well
-known [Gherkin BDD DSL](https://cucumber.io/docs/reference) and
-inspired by solutions built on it, such
-as [cucumber-js](https://github.com/cucumber/cucumber-js).  All logic
-is coded in TypeScript or JavaScript. If you are familiar with
-Cucumber (versions of which exist for many languages), you should find
-the Rug test framework particularly easy to learn; if not, it should
-still be intuitive.
+The framework is based on the well known [Gherkin BDD DSL][gherkin]
+and inspired by solutions built on it, such
+as [cucumber-js][cucumber].  All logic is coded in TypeScript or
+JavaScript. If you are familiar with Cucumber (versions of which exist
+for many languages), you should find the Rug test framework
+particularly easy to learn; if not, it should still be intuitive.
+
+[gherkin]: https://cucumber.io/docs/reference (Gherkin BDD)
+[cucumber]: https://github.com/cucumber/cucumber-js (Cucumber.js)
+
+## Approach
+
+Rug is designed to support [Test Driven Development][tdd] using unit
+tests and BDD.  We recommend following the "red &rarr; green &rarr;
+refactor" approach.
+
+!!! missing "Prerequisites"
+    You will need the [Rug CLI][cli-install], [Node.js][node],
+    and [Yarn][yarn] installed to run tests.
+
+[tdd]: https://en.wikipedia.org/wiki/Test-driven_development (Test-Driven Development)
+[cli-install]: /user-guide/interfaces/cli/install.md (Rug CLI Installation)
+[node]: https://nodejs.org/ (Node.js)
+[yarn]: https://yarnpkg.com/ (Yarn Package Manager)
+
+For unit testing, we recommend using the [Mocha][mocha] framework with
+either [Power Assert][power-assert] or [Chai][chai].  You should put
+your Mocha unit tests in the `.atomist/mocha` directory to avoid
+conflicts with where the Rug CLI looks for the BDD tests.  With this
```

Comment:
this is unfortunate and maybe something we should consider a bug?

I'd rather all tests where under the `tests` directory. Even if that is `tests/mocha`;